kong/db/schema/metaschema.lua

```diff
@@ -393,6 +393,18 @@ local MetaSchema = Schema.new({
         default = true,
       },
     },
+    {
+      path = {
```

Comment:
I suggest renaming `path` and `nested_path`, because it's not clear those are related to Admin API endpoints, plus these are not paths, but path segments. My suggestion is names using `endpoint_*`, to align them with the existing `endpoint_key`.

* `path` → `endpoint_collection_name`
* `nested_path` →`endpoint_entity_name`

It's verbose, I know, but we or the users won't writing those things all the time (and the alternative of using `admin_api_` instead of `endpoint_` was even longer :) ).

Reply:
A follow-up to this based on our out-of-band conversation: looks like the `nested_path` is currently used for both entities (e.g. `GET /consumers/:consumers/key-auth/:keyauth_credentials`) and collections (e.g.  GET `/consumers/:consumers/key-auth`). My other two concerns remain (it's not clear those are related to Admin API endpoints, plus these are not paths, but path segments).

In these case, I sugest these uncool but technically correct names:

* `path` → `admin_api_name` (as in "the name this entity appears as in the Admin API") 
* `nested_path` → `admin_api_nested_name` (as in this entity appears as when nested in the Admin API")

(I avoided `endpoint_name` to prevent confusion with `endpoint_key` — but note that all `*_key` schema attributes refer to field names).

With this suggested rename, I think this PR can be merged.

Reply:
They are not `path segments`path segment is `/<path_segment>/<second-path-segment>`, `path` and `nested_path` can be `something/:lapis_param/anything`.

Nested path is not used for both entities and collections (it always fits to position of `collection`:
```
/consumers/:consumers/key-auth/:keyauth_credentials
```
that can be translated to:
```
/collection/:entity/colection/:entity
```
that can be translated to:
```
/<path_or_name>/:entity/<nested_path_or_path_or_name>/:entity
```

Sure. after collection there can be `entity` endpoint (and this forth level adds one more entity level).

`nested` can also be seen `referenced`, e.g. `consumers` are referenced from `keyauth_credentials`.

I will do the rename, as the suggested names are ok with me.

Reply:
The branch is now rebased (on latest `next`), and `path` is renamed to `admin_api_name` and `nested_path` is renamed to `admin_api_nested_name`.